suse_migration_services/defaults.py

```diff
@@ -75,3 +75,9 @@ def _get_ssh_keys_path(self, prefix_path):
         return os.sep.join(
             [self.get_system_root_path(), prefix_path, '.ssh/authorized_keys']
         )
+
+    @classmethod
+    def get_system_ssh_host_keys_glob_path(self):
+        return os.sep.join(
+            [self.get_system_root_path(), 'etc/ssh/ssh_host_*']
+        )
```

Comment:
The process here is not fail safe. We do a glob match and the files gets copied 1:1 to the live migration system. Whether or not those ssh host keys will be really used depends on the setup in ```/etc/ssh/sshd_config```. As we go with the default setup sshd comes with a prebuilt list of ssh host keys that gets read. This list can be different compared to sles12 vs. sles15. So the concept here should be changed in a way that we copy all existing host keys and additionally add an entry of the form:

```
HostKey /path/to/key
```

to the private host key(s) that was found into ```/etc/ssh/sshd_config```